pypesto/petab/importer.py

```diff
@@ -417,6 +424,31 @@ def create_objective(
             # FIXME: currently not supported with hierarchical
             kwargs['guess_steadystate'] = False
 
+        if self._ordinal:
+            # TODO add constants to C
+            if 'inner_problem_method' in kwargs:
+                inner_problem_method = kwargs.pop('inner_problem_method')
```

Comment:
is popping necessary? usually unnecessary

Reply:
As the kwargs are later passed to `AmiciObjective` and it doesn't take any kwargs, it fails there if they're not popped.